Wireframe/css/style.css

```diff
@@ -0,0 +1,63 @@
+/* General page styling */
+body {
```

Comment:
You have both `padding: 0` and `padding-bottom: 50px` set in the body. The general `padding: 0` will override the specific `padding-bottom`.